matchbook-sdk-rest/src/main/java/com/matchbook/sdk/rest/dtos/events/EventRequest.java

```diff
@@ -55,41 +54,54 @@ public String toString() {
                 ", includeEventParticipants=" + includeEventParticipants +
                 ", includePrices=" + includePrices +
                 (includePrices ? (
-                    ", oddsType=" + oddsType +
-                    ", exchangeType=" + exchangeType +
-                    ", side=" + side +
-                    ", currency=" + currency +
-                    ", minimumLiquidity=" + minimumLiquidity +
-                    ", priceMode=" + priceMode
+                        ", oddsType=" + oddsType +
+                                ", exchangeType=" + exchangeType +
```

Comment:
Are we sure about these white spaces?

Reply:
i'm not sure what do you mean

Reply:
The extra and not consistent white spaces at the beginning of the line.